wimprates/halo.py

```diff
@@ -84,36 +89,48 @@ def earth_velocity(t):
     v_earth_sun = v_orbit * (e_1 * np.cos(phi) + e_2 * np.sin(phi))
 
     # Velocity of Local Standard of Rest
-    v_lsr = np.array([0, 220, 0]) * nu.km/nu.s
+    v_lsr = np.array([0, v_0/(nu.km/nu.s), 0]) * nu.km/nu.s
```

Comment:
This can just be v_lsr = np.array([0, v0, 0]), right? No need to divide and mulitply by the units.

Reply:
Yes thanks! I thought we'd have some artifact from `nu` when we change units but that isn't true